torch/csrc/api/include/torch/nn/modules/embedding.h

```diff
@@ -12,20 +12,32 @@ namespace nn {
 
 /// Options for the `Embedding` module.
 struct TORCH_API EmbeddingOptions {
-  EmbeddingOptions(int64_t count, int64_t dimension);
-  /// The number of embeddings (number of rows in the table).
-  TORCH_ARG(int64_t, count);
-  /// The size of each embedding vector (number of columns in the table).
-  TORCH_ARG(int64_t, dimension);
+  EmbeddingOptions(int64_t num_embeddings, int64_t embedding_dim);
+  // The number of embeddings (number of rows in the table).
+  TORCH_ARG(int64_t, num_embeddings);
+  // The size of each embedding vector (number of columns in the table).
+  TORCH_ARG(int64_t, embedding_dim);
+  // If given, pads the output with the embedding vector at :attr:`padding_idx (initialized to zeros) whenever it encounters the index.
```

Comment:
`:attr:` probably doesn't work in C++ API docs now, and we can change it to:
```suggestion
  // If given, pads the output with the embedding vector at `padding_idx` (initialized to zeros) whenever it encounters the index.
```